cli/src/remote/watcher.js

```diff
@@ -120,10 +120,12 @@ export default class RemoteWatcher {
       return this.prep.deleteDocAsync(SIDE, was)
     } else {
       const {path} = doc
-      if (['directory', 'file'].includes(doc.type)) {
-        return this.putDoc(doc, was)
-      } else {
+      if (doc.type !== 'directory' && doc.type !== 'file') {
         log.error({path}, `Document ${doc._id} is not a file or a directory`)
+      } else if (doc.type === 'file' && doc.md5sum == '') {
```

Comment:
I believe the `md5sum` field will be missing for temporary files, it won't contain an empty string.

Sample error from logs:

```
20:49:04.496Z  WARN Cozy Desktop/Metadata:
    Invalid checksum: {
      "path": "Photos/Transférées depuis Cozy Photos/Capture d’écran 2017-07-21 à 22.47.59.png",
      "docType": "file",
      "updated_at": "2017-07-21T20:48:40.465257221Z",
      "remote": {
        "_id": "ce5cba86e5117eb1e5aa603b534ab2e6",
        "_rev": "1-fa6493586e0d9cc5caab88a5d1b11454"
      },
      "size": 103790,
      "class": "image",
      "mime": "image/png",
      "tags": []
    }
    --
    path: Photos/Transférées depuis Cozy Photos/Capture d’écran 2017-07-21 à 22.47.59.png
```